src/pip/_internal/commands/download.py

```diff
@@ -132,19 +131,12 @@ def run(self, options, args):
                     download_dir=options.download_dir,
                 )
 
-                resolver = Resolver(
+                resolver = self.make_resolver(
                     preparer=preparer,
-                    finder=finder,
```

Comment:
nit: I would keep finder in the second position in all of these to make the diff cleaner.

Reply:
I just wanted to keep the ordering from Resolver class.